src/components/menu/menu.xml

```diff
@@ -18,7 +18,7 @@
               t-att-data-name="menuItem.id"
               t-on-click="(ev) => this.onClickMenu(menuItem, menuItem_index, ev)"
               t-on-mouseover="(ev) => this.onMouseOver(menuItem, menuItem_index, ev)"
-              class="o-menu-item"
+              class="o-menu-item d-flex align-items-center"
```

Comment:
The justify-content: space-between is not present anymore. Is it intended ?

Reply:
Yeah it was useless, it's kinda overwritten by the `ms-auto` of the children